src/MenuGroup/index.web.js

```diff
@@ -0,0 +1,4 @@
+// @flow
+
+export const MenuGroup = () => 'Not yet implemented';
```

Comment:
Could you instead say: `Currently only available on Android and iOS`

Reply:
Sure